API/Controllers/ProjectController.cs

```diff
@@ -239,6 +239,35 @@ public async Task<IActionResult> MigrateDatabaseToElasticSearch()
             return Ok(resultsResource);
         }
 
+        /// <summary>
+        ///     This method returns suggestions while searching for projects
+        /// </summary>
+        /// <param name="query"></param>
+        /// <returns>This method returns a list of autocomplete project resources.</returns>
+        /// <response code="200">This endpoint returns a list with suggested projects.</response>
+        [HttpGet("search/autocomplete")]
+        [ProducesResponseType(typeof(List<AutocompleteProjectResourceResult>), (int) HttpStatusCode.OK)]
+        public async Task<IActionResult> GetAutoCompleteProjects([FromQuery(Name ="query")] string query)
+        {
+            try
+            {
+                List<Project> projects = await projectService.FindProjectsWhereTitleStartsWithQuery(query);
+                List<AutocompleteProjectResourceResult> autocompleteProjectResourceResults = mapper.Map<List<Project>, List<AutocompleteProjectResourceResult>>(projects);
+                return Ok(autocompleteProjectResourceResults);
+            }
+            catch(Exception)
+            {
+                return BadRequest(
+                    new ProblemDetails
+                    {
+                        Title = "Autocomplete results could not be retrieved.",
+                        Detail = "Something went wrong.",
```

Comment:
I think 'Something went wrong.' is kind of vague. I think it's ok to keep things somewhat vague for the end-user, but I think this is a step too far. In my opinion, this can be a little bit more specific.

Reply:
I also agree to change the description.